CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # bedrock-validation ChangeLog
 
+## 7.1.0 -
+
+### Added
+- Added a validator for `VerifiablePresentation`
+- Added a validator for `VerifiableCredential`.
+- Added a validator for `jsonldIdentifier`.
```

Comment:
```suggestion
```

Reply:
I addressed this here: https://github.com/digitalbazaar/bedrock-validation/pull/44/commits/826e648f8441decfc6566128e796e3bc4eb89a58